packages/ai-constructs/src/conversation/runtime/bedrock_converse_adapter.test.ts

```diff
@@ -451,396 +843,6 @@ void describe('Bedrock converse adapter', () => {
     );
   });
 
-  void it('executable tool error is reported to bedrock', async () => {
```

Comment:
Some tests are moved up and down to accommodate new grouping. They're NOT deleted.